nixos/modules/services/networking/unbound.nix

```diff
@@ -14,6 +14,18 @@ let
 
   isLocalAddress = x: substring 0 3 x == "::1" || substring 0 9 x == "127.0.0.1";
 
+  remote =
+    optionalString (cfg.enableRemoteAccess) ''
+    remote-control:
+      control-enable: "yes"
+      server-key-file: ${stateDir}/unbound_server.key
+      server-cert-file: ${stateDir}/unbound_server.pem
+      control-key-file: ${stateDir}/unbound_control.key
+      control-cert-file: ${stateDir}/unbound_control.pem
```

Comment:
Are these going to be correctly auto-gen'd when the cfg is enabled?

Reply:
They should be with this :+1: https://github.com/NixOS/nixpkgs/compare/master...kraem:nixos/unbound?expand=1#diff-af0e676aa026493eac8384bd688abbdfR191

Reply:
Just verified this since it was a long time since I wrote this and it works for me :+1: